DEVELOPING.md

```diff
@@ -38,6 +38,16 @@ to a gcr.io repository before the integration tests can run.
 ```bash
 $ RUNTIME_IMAGE=gcr.io/my-project-id/openjdk:my-tag
 $ gcloud docker -- push $RUNTIME_IMAGE
-$ ./scripts/integration_test.sh $RUNTIME_IMAGE
 ```
 
+You have the possibility to either run the tests on App Engine or on Google Container Engine.
```

Comment:
Then here just say that you have the option to run the AE and GKE tests individually like this: